src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <ul class="calendar calendar__start-day--fri calendar__month-length--30">
+      <li class="calendar__day calendar__day--1"></li>
```

Comment:
You don't need to specify separate class for each `li` element with `li` number
Use `nth-child` instead of separate class